applications/TopologyOptimizationApplication/examples/00_Small_Cantilever_Hexahedral_Mini/run_TopOpt.py

```diff
@@ -0,0 +1,129 @@
+#import kratos core and applications
+import KratosMultiphysics as km
+import KratosMultiphysics.StructuralMechanicsApplication as ksm
+import KratosMultiphysics.TopologyOptimizationApplication as kto
+import KratosMultiphysics.LinearSolversApplication as kls
+import os
+import OptimizationParameters as opt_parameters
```

Comment:
i suggest to store the opt settings also as a json file, like it is done in the rest of Kratos